_data/assignments.yaml

```diff
@@ -101,17 +101,17 @@ labs:
       visible: true
   -
       name: "Ideal Indirection"
-      releaseDate: "Week 8"
-      dueDate: "Week 9"
+      releaseDate: "Week 9"
+      dueDate: "Week 10"
```

Comment:
Good catch! We will have to update this for every subsequent lab release this semester, until the one we drop.